spec/uri_parser_spec.rb

```diff
@@ -21,18 +21,18 @@ def self.describe_parsed(url, &block)
     its(:host) { should == 'example.com' }
     its(:path) { should == '/foo/bar' }
     its(:query) { should == 'a=b&c=d' }
-    its(:valid?) { should be_true }
+    its(:valid?) { should be_truthy }
```

Comment:
Why this change?

Reply:
Required change due to updated rspec version. The updated rspec version was required to work with Ruby 2.